### PR TITLE
add retry + send mail

### PR DESCRIPTION
**Pull request by flavienLh**

### CODE REVIEW

File: snippet_project/settings.py [hunk 1]
```diff
@@ -144,27 +140,9 @@
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
 
 # Media Configuration
-IS_PRODUCTION = os.getenv("DJANGO_PRODUCTION", "False") == "True"
```
Comment (CharlyRousseau) [inline snippet_project/settings.py]: Why delete this ?

File: snippet_project/settings.py [hunk 2]
```diff
@@ -178,20 +156,6 @@
 handler404 = "snippet.view.custom_page_not_found"
 handler500 = "snippet.view.custom_internal_error"
 
-LOGGING = {
```
Comment (CharlyRousseau) [inline snippet_project/settings.py:181]: Also this we need to log the 500 errors  in the pod

File: snippet_project/settings.py [hunk 3]
```diff
@@ -80,23 +87,12 @@
 # Database
 # https://docs.djangoproject.com/en/5.0/ref/settings/#databases
 
-DATABASE_URL = os.getenv("DATABASE_URL")
```
Comment (CharlyRousseau) [inline snippet_project/settings.py]: Good idea to delete prod database setting 👍

File: snippet_project/settings.py [hunk 4]
```diff
@@ -23,7 +24,6 @@
 
 ALLOWED_HOSTS = ["snippet.playground-charly.fr", "127.0.0.1", "localhost"]
```
Comment (Snowsdy) [inline snippet_project/settings.py]: Tu le rajoutes

File: snippet_project/settings.py (hunk 3 above)
Comment (Snowsdy) [inline snippet_project/settings.py]: Tu rajoutes tout le bloc supprimé

File: snippet_project/settings.py (hunk 1 above)
Comment (Snowsdy) [inline snippet_project/settings.py]: Idem, tu reajoutes tout le block supprimé

File: snippet_project/settings.py (hunk 2 above)
Comment (Snowsdy) [inline snippet_project/settings.py:181]: Idem

**Review by Snowsdy — CHANGES_REQUESTED** [review]

File: requirements.txt [hunk 5]
```diff
@@ -55,3 +55,5 @@ tqdm==4.66.4
 typing_extensions==4.12.2
 urllib3==2.2.2
 yarl==1.9.4
+gunicorn
```
Comment (CharlyRousseau) [inline requirements.txt]: no needed already in it

**Review by CharlyRousseau — CHANGES_REQUESTED** [review]

**Review by Snowsdy — APPROVED** [review]

**Review by Snowsdy — APPROVED** [review]

**Pull request closed without merging**